SocialStudies/OrphanWells/orphan-wells.ipynb

```diff
@@ -112,10 +112,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import warnings, requests\n",
     "url = 'https://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'\n",
     "\n",
-    "import pandas as pd\n",
-    "ww = pd.read_html(url)[1]\n",
+    "#import ssl\n",
```

Comment:
Should probably delete these commented/unused lines

Reply:
I thought there were put there intentionally for having different options. I will remove them.

Reply:
I don't think so... I think we're good to just get rid of them.